### DISCUSSION TITLE
Any directory should hide some unnecessary files.

### DISCUSSION THREAD

**Discussion by noa7777** [gsantner/markor#2415] [Ideas] [answered]
There are a lot of pictures in the catalog, but they are just attachments. I think these pictures should be hidden. I wish there was a button to set them.

**Comment by harshad1** [comment]
You can have attachments in a subfolder. I like _res but .res is hideable

**Reply by noa7777** [reply]
Thanks for the answer. I can try this.

**Comment by gsantner** [comment, marked as answer]
Prefixing the file with a dot, or putting it in a subfolder should help. This also applies to most other file management software and is not Markor specific.

**Reply by noa7777** [reply]
No, no, no. This will be very troublesome. Usually, when I insert an image, I will select the image on my mobile phone, and the image will be automatically cloned into this directory. It is very troublesome to add "."in front of the one you are talking about.

**Reply by gsantner** [reply]
@noa7777 
You got replies by two people and have two suggestions, one works without manual action everytime.

Please accept that Markor can't please everybody to 100% personal needs. There are other users as well, and what you might consider neccassary is bloat to them they want to turn off.

**Reply by noa7777** [reply]
Thank you for your answer. I think the attachments folder can be tried, although it is a bit troublesome.
